ethereum/datastructures/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
   implementation project(':logging')
   implementation project(':pow')
   implementation project(':util')
+  implementation project(':storage:api')
```

Comment:
I don't think we should have this dependency.  I suspect `SlotAndBlockRoot` should just be in `:ethereum:datastructures`